deploy/iso/minikube-iso/package/automount/minikube-automount

```diff
@@ -130,6 +130,9 @@ if [ -n "$BOOT2DOCKER_DATA" ]; then
     mkdir -p /mnt/$PARTNAME/hostpath_pv
     ln -s /mnt/$PARTNAME/hostpath_pv /tmp/hostpath_pv
 
+    mkdir -p /mnt/$PARTNAME/hostpath-provisioner
+    ln -s /mnt/$PARTNAME/hostpath_pv /tmp/hostpath-provisioner
```

Comment:
Copy typo `/mnt/$PARTNAME/hostpath-provisioner` instead of `/mnt/$PARTNAME/hostpath_pv`.